addons/website/views/snippets/s_dynamic_snippet.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <template id="website.s_dynamic_snippet_template">
+        <!-- TODO In master, replace `d-none` by `o_dynamic_empty` -->
```

Comment:
Indeed, to be done in master. But in stable, you can already patch the `onBuilt` of the related option to replace d-none by the right class once the snippet is added in the DOM (best not wait for the public widget to do it (even though still necessary to do it there for compatibility), `onBuilt` should normally be safer regarding editor history etc).